developer-experience/recipes/bridge-to-kubernetes.md

```diff
@@ -0,0 +1,60 @@
+
+# Bridge to Kubernetes
+
+Bridge to Kubernetes allows developers to seamlessly work on services both in a shared cluster and locally in debug mode.
+
+To learn more about Bridge to Kubernetes:
+
+- [Introduction](https://code.visualstudio.com/docs/containers/bridge-to-kubernetes)
+
+- [How it works](https://docs.microsoft.com/en-us/visualstudio/containers/overview-bridge-to-kubernetes)
+
+- [Configuration](https://docs.microsoft.com/en-us/visualstudio/containers/configure-bridge-to-kubernetes)
+
+- [VS Code Setup](https://docs.microsoft.com/en-us/visualstudio/containers/bridge-to-kubernetes?view=vs-2019)
+
+- [Repository](https://github.com/Microsoft/mindaro)
+
+## Useful Scenarios
+
+We believe this technology can be most helpful...
+
+- if a developer needs to test/debug a new or changed service against an ecosystem of services.
+
+- if a developer does not have the domain knowledge to deploy all the micro services. This allows them to work against a "blackbox" set of already deployed services.
+
+We found the technology very easy to implement so it can probably be used for a wide of cases with minimal training.
+
+## Limitations
+
+While many of these limitations are in the backlog, currently, you cannot use Bridge to Kubernetes in these circumstances...
```

Comment:
nit: remove the ...

Reply:
Also, I would recommend removing this section all together and putting it in the bridge to kubernetes repo instead, otherwise you have to keep updating it in two places or this will become stale as soon as it is updated - Keep it DRY

Reply:
I added:
Please check [here](https://docs.microsoft.com/en-us/visualstudio/containers/overview-bridge-to-kubernetes?view=vs-2019#limitations) for updated limitations.

I don't maintain the bridge-to-kubernetes repo, but I still think think it is helpful to at least be aware there may be significant limitations and then they can follow the link and check.